### ISSUE TITLE
Export issue

### ISSUE THREAD

**Issue by EDIflyer** [authenticatorplus/logos#267]
Sorry to contact you via GitHub @MufriA but I tried via the website and didn't hear back. Export is failing, just giving an error - do you have a workaround?

**Comment by BitPopCoin** [comment]
Seems to be broken in later androids. Developer is MIA for a while now sadly.

**Comment by EDIflyer** [comment]
Yep, thankfully managed to export on an older phone and got them all across into Bitwarden. Shame as its still a great app.
